src/commands/version.cr

```diff
@@ -0,0 +1,28 @@
+require "../helpers/versions"
+require "./command"
+
+module Shards
+  module Commands
+    class Version < Command
+      def self.run(path, *args)
+        # look up for `shard.yml` in given directory or up
+        previous = nil
+        current = File.expand_path(path)
+
+        until !File.directory?(current) || current == previous
+          shard_file = File.join(current, SPEC_FILENAME)
+          break if File.exists?(shard_file)
+
+          previous = current
+          current = File.expand_path("..", current)
+        end
+
+        new(current).run(*args)
+      end
+
+      def run(*args)
+        puts spec.version
```

Comment:
@RX14 for the reason given by @Sija:

```
$ shards version
0.3.7
$
```

Using `print` breaks the resulting prompt.